libs/providers/launchdarkly-client/README.md

```diff
@@ -0,0 +1,27 @@
+# launchdarkly-client Provider
+
+## Installation
+
+```
+$ npm install @openfeature/launchdarkly-client-provider
+```
+
+## Building
+
+Run `nx package providers-launchdarkly-client` to build the library.
+
+## Running unit tests
+
+Run `nx test providers-launchdarkly-client` to execute the unit tests via [Jest](https://jestjs.io).
+
+## Sample initialization
+``` ts
+import {initialize} from 'launchdarkly-js-client-sdk'; 
+
+const initialContext = {};
+const ldClient =  initialize('LDId', initialContext);
+await ldClient.waitForInitialization()
+const ldOpenFeatureProvider = new LaunchDarklyClientProvider(ldClient);
+
+OpenFeature.setProvider(ldOpenFeatureProvider);
+```
```

Comment:
I think we should to rigorously document some of the context-related stuff here, both initialization and updates. For example one thing to note is that updating the context will cause the `identify` function to run, which results in network traffic (correct me if I'm wrong @kinyoklion ).

Reply:
That is correct. The nature of the JS client SDK is such.

1. Do a `GET` or a `REPORT` to get the current flags for the identified context.
2. If the stream option is set, or someone registers a flag listener, or they call the `stream` method, then a stream is initiated.
3. The stream remains open getting updates to flag values for that context.

Whenever you identify this process restarts. So we only want to identify if the context is actually different.

Versus our server SDKs which download entire rulesets, and evaluate all contexts against that ruleset. On those SDKs identify also exists, but all it does is send an analytics event so the context can be presented in our UI.

(Currently OF doesn't have listeners, so that is one of the reasons to have direct client access.)